From e3827eb52b00c700d2830918aca8a00f9e4f1f02 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Tue, 19 May 2015 21:20:44 +0200
Subject: [PATCH 156/310] tmsnc: delete package

Microsoft switched off MSN servers on 31. October 2014
upstream dead/discontinued since 2007

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/tmsnc/Makefile | 49 -------------------------------------------------
 1 file changed, 49 deletions(-)
 delete mode 100644 net/tmsnc/Makefile

diff --git a/net/tmsnc/Makefile b/net/tmsnc/Makefile
deleted file mode 100644
index b7b25fc..0000000
--- a/net/tmsnc/Makefile
+++ /dev/null
@@ -1,49 +0,0 @@
-#
-# Copyright (C) 2006 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=tmsnc
-PKG_VERSION:=0.3.2
-PKG_RELEASE:=1
-
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/tmsnc
-PKG_MD5SUM:=337dae91d4775e03b6681f61db56d2a7
-
-PKG_INSTALL:=1
-
-include $(INCLUDE_DIR)/package.mk
-
-define Package/tmsnc
-  SUBMENU:=Instant Messaging
-  SECTION:=net
-  CATEGORY:=Network
-  DEPENDS:=+libncurses +libopenssl +libpthread
-  TITLE:=text-based MSN client
-  URL:=http://tmsnc.sourceforge.net
-endef
-
-define Package/tmsnc/description
-	TMSNC is a textbased (console) MSN client written in C.
-endef
-
-define Build/Configure
-	$(call Build/Configure/Default, \
-		--disable-rpath \
-		--with-openssl="$(STAGING_DIR)/usr" \
-		--with-ncurses="$(STAGING_DIR)/usr" \
-		--without-libiconv-prefix \
-	)
-endef
-
-define Package/tmsnc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/tmsnc $(1)/usr/bin/
-endef
-
-$(eval $(call BuildPackage,tmsnc))
-- 
2.5.0

